### ISSUE TITLE
Dev and test playload script without flashing

### ISSUE THREAD

**Issue by kar2013** [brandonlw/Psychson#108]
Hi,

first of all thanks for this great work.

My question is very simple, i' want to develop my own playload scripts but i want to know if it's possible to test them without flasing any time my usb device ?

thanks for your answer,

ps: sorry for my english

**Comment by RedToor** [comment]
yes, just change extention of file for .vbs, the payloads is written in Visual basic script.

**Comment by kar2013** [comment]
Hi ReToor,

Thanks for your answer.

I try your suggestion but it's don't work.
 i use this code in vbs file :

 DELAY 3000
GUI r
DELAY 200
STRING notepad
ENTER

Thanks for your help
